test/console/compliance/clusters_test.exs

```diff
@@ -0,0 +1,24 @@
+defmodule Console.Compliance.Datasource.ClustersTest do
+  use Console.DataCase, async: true
+
+  test "it includes read and write users in the compliance report" do
+    user1 = insert(:user, email: "user1@example.com")
+    user2 = insert(:user, email: "user2@example.com")
+    group = insert(:group, name: "compliance-group")
+
+    cluster = insert(:cluster)
```

Comment:
fwiw, a shorter way to set up the bindings here is:

```elixir
cluster = insert(:cluster, read_bindings: [%{user_id: user1.id}, %{group_id: group.id}], write_bindings: [%{user_id: user2.id}])
```

(most of the existing tests do that, and most associations can be instantiated similarly using the testdata lib)